examples/webgl_loader_vrml.html

```diff
@@ -31,23 +31,23 @@
 			import { OrbitControls } from './jsm/controls/OrbitControls.js';
 			import { VRMLLoader } from './jsm/loaders/VRMLLoader.js';
 
-			var camera, scene, renderer, stats;
+			var camera, scene, renderer, stats, controls;
 
 			init();
 			animate();
 
 			function init() {
 
 				camera = new THREE.PerspectiveCamera( 60, window.innerWidth / window.innerHeight, 0.1, 1e10 );
-				camera.position.z = 6;
+				camera.position.set( - 10, 5, 10 );
 
 				scene = new THREE.Scene();
 				scene.add( camera );
 
 				// light
 
 				var dirLight = new THREE.DirectionalLight( 0xffffff );
-				dirLight.position.set( 200, 200, 1000 ).normalize();
+				dirLight.position.set( 200, 200, 1000 );
 
 				camera.add( dirLight );
 				camera.add( dirLight.target );
```

Comment:
@Mugen87 For a later PR, this pattern occurs in several examples. I'd remove it... Perhaps replace it with simply `camera.add( pointLight )`?

Reply:
I have no preference there. I just think using directional lights are fine. The call of `normalize()` however is indeed confusing and should be removed 👍